framerail/src/lib/server/deepwell/page.ts

```diff
@@ -63,7 +67,25 @@ export async function pageMove(
     site_id: siteId,
     page: pageId ?? slug,
     new_slug: newSlug,
-    user_id: 1, // TODO: identify user session and pass the user to the API request
+    user_id: userId,
     revision_comments: revisionComments
   })
 }
+
+export async function pageRevision(
+  siteId: number,
+  pageId: Optional<number>,
+  revisionNumber: Optional<number>,
+  compiledHtml: boolean,
+  wikitext: boolean
```

Comment:
If these are optional, then isn't `compiledHtml?: boolean` and `wikitext?: boolean` necessary?